deploy/sample-run-farm-recipes/aws_ec2.yaml

```diff
@@ -0,0 +1,24 @@
+run_farm_type: AWSEC2F1
```

Comment:
I think removing the `sample` bit wold be nice since, unlike the other sample files, which are copied and modified, these will see active and direct use. So maybe: `build-farm-recipes` and `run-farm-recipes`

Reply:
+1 to what @davidbiancolin said